integreat_cms/cms/templates/pages/page_revisions.html

```diff
@@ -81,13 +91,31 @@ <h1>{{ page_translation.title }}</h1>
         </div>
     </div>
-    <div class="w-full p-4 flex justify-end gap-4">
+    <div class="w-full p-4 flex justify-end gap-4 action-buttons">
         {% if not page.archived %}
             {% has_perm 'cms.change_page_object' request.user page as can_edit_page %}
             {% has_perm 'cms.publish_page_object' request.user page as can_publish_page %}
             {% if can_publish_page %}
-                <button name="submit_draft" class="btn btn-gray">{% trans 'Restore this version as draft' %}</button>
-                <button name="submit_public" class="btn">{% trans 'Restore and publish this version' %}</button>
+                <!-- Draft buttons -->
+                <button name="submit_draft" class="btn btn-gray hidden" data-status="{{ PUBLIC }}">{% trans 'Restore as draft' %}</button>
+                <button name="submit_draft" class="btn btn-gray hidden" data-status="{{ DRAFT }}">{% trans 'Restore as draft' %}</button>
+                <button name="submit_draft" class="btn btn-gray hidden" data-status="{{ AUTO_SAVE }}">{% trans 'Restore auto save as draft' %}</button>
+                <button name="submit_draft" class="btn btn-gray hidden" data-status="{{ REVIEW }}">{% trans 'Accept and restore changes as draft' %}</button>
+                <!-- Draft buttons for latest version -->
+                <button name="submit_draft" class="btn btn-gray hidden" data-status="{{ REVIEW }}" data-max>{% trans 'Accept current changes as draft' %}</button>
+                <!-- Publish buttons -->
+                <button name="submit_public" class="btn hidden" data-status="{{ PUBLIC }}">{% trans 'Restore and publish' %}</button>
+                <button name="submit_public" class="btn hidden" data-status="{{ DRAFT }}">{% trans 'Restore and publish' %}</button>
+                <button name="submit_public" class="btn hidden" data-status="{{ AUTO_SAVE }}">{% trans 'Restore and publish auto save' %}</button>
+                <button name="submit_public" class="btn hidden" data-status="{{ REVIEW }}">{% trans 'Accept, restore and publish changes' %}</button>
+                <!-- Publish buttons for latest version  -->
+                <button name="submit_public" class="btn hidden" data-status="{{ PUBLIC }}" data-max>{% trans 'Refresh date' %}</button>
+                <button name="submit_public" class="btn hidden" data-status="{{ DRAFT }}" data-max>{% trans 'Publish the current draft' %}</button>
+                <button name="submit_public" class="btn hidden" data-status="{{ AUTO_SAVE }}" data-max>{% trans 'Publish auto save' %}</button>
+                <button name="submit_public" class="btn hidden" data-status="{{ REVIEW }}" data-max>{% trans 'Accept and publish the current changes' %}</button>
+                <!-- Reject buttons for latest version  -->
+                <button name="submit_reject" class="btn btn-red hidden" data-status="{{ AUTO_SAVE }}" data-max>{% trans 'Discard auto save' %}</button>
```

Comment:
Could you switch the position of the "discard autosave" button? The button breaks our current setup with the submit button always being in the bottom right corner :)

Reply:
Yes, thanks, makes sense! :+1: